apis/dispatch.php

```diff
@@ -407,3 +408,17 @@
     $logger->debug("dispatch.php no route found for resource", ['resource' => $resource]);
     http_response_code(404);
 }
+
+// Report to Telemetry
+// Be nice to find a centralized place to catch failed requests.
```

Comment:
I pull this stuff for the realworldtesting from the api log and log. (can get success, fails, from patient, from users, and resource) I could probably work this into the apiLog() function if you want (ie. put it into your tracking table in apiLog, then can record fails).

Reply:
actually, not seeing fails being logged there. will research a bit more, but may be best to keep here if can't get centralized access to fails in apiLog().

Reply:
Goodness, if above is true, then this means the Reals World Testing apis will never fail :)